pumpkin/src/client/player_packet.rs

```diff
@@ -84,6 +85,22 @@ impl PumpkinError for BlockPlacingError {
     }
 }
 
+static CREATIVE_DONT_BREAK_WITH: LazyLock<HashSet<u16>> = LazyLock::new(|| {
+    [
+        get_item("wooden_sword").unwrap().id,
+        get_item("stone_sword").unwrap().id,
+        get_item("iron_sword").unwrap().id,
+        get_item("golden_sword").unwrap().id,
+        get_item("diamond_sword").unwrap().id,
+        get_item("netherite_sword").unwrap().id,
```

Comment:
Why not just check if it's a sword instead of having a hardcoded list of items?

Reply:
is_sword is also hardcoded, and it is missing the trident and mace

Reply:
> is_sword is also hardcoded, and it is missing the trident and mace

:facepalm:
something filtering over all items could filter using `contains` and checking if the item name contains `sword`
```rs
itemName.contains("sword")
```
Also, doesn't Minecraft have metadata for which category or what type of item this is (e.g. swords)?